### ISSUE TITLE
Formatting and file case sensitivity

### ISSUE THREAD

**Issue by MellowSwell** [BtheDestroyer/Godot-YASM#1]
Awesome plugin, thanks for making this available! Some feedback/issues:

Whatever is going on with the indentation here, the standard Godot 4.2 engine absolutely hates it. Seems like you might be using smaller than normal tabs or just using spaces. Either way, I ended up having to go through and manually fix the indentation with standard tabs for Godot to stop throwing a fit.

Also according to the best practices I'd suggest fixing your file naming scheme to match godot best practices since it also likes to throw a fit and create weird dupes if you have capitalized file names in Windows. (https://docs.godotengine.org/en/stable/tutorials/best_practices/project_organization.html#case-sensitivity)

**Comment by Bryce-Dixon** [comment]
> Whatever is going on with the indentation here, the standard Godot 4.2 engine absolutely hates it. Seems like you might be using smaller than normal tabs or just using spaces

Yeah... I wish Godot allowed for plugin-specific indentation settings...
I prefer 2 space indentation over 4-width tabs because I feel tabs appearing differently on different machines leads to weird line-width issues and 4 spaces is a bit excessive. Everyone has their own preference, so it's frustrating that Godot's editor doesn't account for that when importing files (I frequently have to manually adjust tabs since Godot automatically converts indentation by default).

Maybe I can take a look at GitHub Actions for the first time to convert the indentation to engine defaults before generating a release?

> Also according to the best practices I'd suggest fixing your file naming scheme to match godot best practices since it also likes to throw a fit and create weird dupes if you have capitalized file names in Windows.

That's strange; I primarily develop on Windows 10 (though I also use Ubuntu sometimes) and have never seen this issue. Regardless, yes I can and probably should update the naming convention.
